src/modules/colorPicker/ColorPickerUI/Controls/ColorPickerControl.xaml

```diff
@@ -213,19 +212,27 @@
                 Style="{DynamicResource ColorShadeButtonStyle}"
                 ToolTipService.ToolTip="{x:Static p:Resources.Selected_color_tooltip}">
 
-                <ui:Flyout
+                <Popup
                     x:Name="DetailsFlyout"
                     Margin="24,0,0,0"
                     Closed="DetailsFlyout_Closed"
                     Opened="DetailsFlyout_Opened"
+                    AllowsTransparency="True"
+                    StaysOpen="False"
                     Placement="Top">
-                    <Grid x:Name="detailsGrid" KeyboardNavigation.TabNavigation="Contained">
-                        <Grid.ColumnDefinitions>
+                    <Border
+                        CornerRadius="10"
```

Comment:
IT is generally recommended to always use multiples of four - because that's best for scaling factors. Also, there are resources for this. See WinUI Gallery.